scripts-python/case.build

```diff
@@ -0,0 +1,508 @@
+    os_                 = case.get_value("OS")
+    comp_cpl            = case.get_value("COMP_CPL")
+    machines_file       = case.get_value("MACHINES_SPEC_FILE")
+    ocn_submodel        = case.get_value("OCN_SUBMODEL")
+    profile_papi_enable = case.get_value("PROFILE_PAPI_ENABLE")
+    nthrds_cpl          = int(case.get_value("NTHRDS_CPL"))
+    nthrds_atm          = int(case.get_value("NTHRDS_ATM"))
+    nthrds_lnd          = int(case.get_value("NTHRDS_LND"))
+    nthrds_ice          = int(case.get_value("NTHRDS_ICE"))
+    nthrds_ocn          = int(case.get_value("NTHRDS_OCN"))
+    nthrds_glc          = int(case.get_value("NTHRDS_GLC"))
+    nthrds_wav          = int(case.get_value("NTHRDS_WAV"))
+    nthrds_rof          = int(case.get_value("NTHRDS_ROF"))
+
+    # Load some params into env
+    os.environ["CIMEROOT"]             = cimeroot
+    os.environ["CASETOOLS"]            = casetools
+    os.environ["EXEROOT"]              = exeroot
+    os.environ["INCROOT"]              = incroot
+    os.environ["LIBROOT"]              = libroot
+    os.environ["SHAREDLIBROOT"]        = sharedlibroot
+    os.environ["CASEROOT"]             = caseroot
+    os.environ["COMPILER"]             = compiler
+    os.environ["COMP_INTERFACE"]       = comp_interface
+    os.environ["NINST_VALUE"]          = ninst_value
+    os.environ["BUILD_THREADED"]       = build_threaded
+    os.environ["MACH"]                 = mach
+    os.environ["USE_ESMF_LIB"]         = use_esmf_lib
+    os.environ["MPILIB"]               = mpilib
+    os.environ["DEBUG"]                = debug
+    os.environ["OS"]                   = os_
+    os.environ["COMP_CPL"]             = comp_cpl
+    os.environ["COMP_ATM"]             = comp_atm
+    os.environ["COMP_LND"]             = comp_lnd
+    os.environ["COMP_ICE"]             = comp_ice
+    os.environ["COMP_OCN"]             = comp_ocn
+    os.environ["COMP_GLC"]             = comp_glc
+    os.environ["COMP_WAV"]             = comp_wav
+    os.environ["COMP_ROF"]             = comp_rof
+    os.environ["CLM_CONFIG_OPTS"]      = clm_config_opts     if clm_config_opts     is not None else ""
+    os.environ["CAM_CONFIG_OPTS"]      = cam_config_opts     if cam_config_opts     is not None else ""
+    os.environ["PIO_CONFIG_OPTS"]      = pio_config_opts     if pio_config_opts     is not None else ""
+    os.environ["OCN_SUBMODEL"]         = ocn_submodel        if ocn_submodel        is not None else ""
+    os.environ["PROFILE_PAPI_ENABLE"]  = profile_papi_enable if profile_papi_enable is not None else ""
+    os.environ["CLM_USE_PETSC"]        = clm_use_petsc       if clm_use_petsc       is not None else ""
+    os.environ["CISM_USE_TRILINOS"]    = cism_use_trilinos   if cism_use_trilinos   is not None else ""
+    os.environ["MPASLI_USE_ALBANY"]    = mpasli_use_albany   if mpasli_use_albany   is not None else ""
+    lid                                = run_cmd("date +%y%m%d-%H%M%S") # why not testid?
```

Comment:
This is a timestamp for the build , not the same as the testid, and this case may not be a test anyway.   For a production experiment there may be many builds of the same case.